doc/nrf/ug_nrf9160.rst

```diff
@@ -199,8 +201,66 @@ Likewise, in Step 7, choose the :file:`.hex` file for the application you are pr
 .. note::
    When you update the application firmware on an nRF9160 DK, you must also update the modem firmware as described in :ref:`nrf9160_gs_updating_fw_modem`.
 
-See :ref:`gs_programming` for more information on building and programming samples on nRF9160 DK.
+.. _build_pgm_nrf9160_vsc:
+
+Building and programming using |VSC|
+====================================
+
+|vsc_extension_instructions|
+
+Complete the following steps after installing the |nRFVSC|:
+
+.. |sample_path_vsc| replace:: :file:`samples/nrf9160/aws_iot`
+
+.. |vsc_sample_board_target_line| replace:: you must use the build target ``nrf9160dk_nrf9160`` or ``nrf9160dk_nrf9160_ns`` when building the application code for the nRF9160 DK
```

Comment:
I think we should remove `nrf9160dk_nrf9160` for this guide. It's very rare a customer would use it, so it's probably better to document those few use-cases in a separate guide.

Reply:
Does any of the samples not work for the `nrf9160dk_nrf9160_ns` board? Even if a few samples that doesn't use the modem can work on the nrf9160, the feature they're demonstrating wouldn't work in a real life application that uses the modem. So I disagree we should keep the secure board in the general user guide. I think the only real reason to use the secure board if is you're building your own alternative to TF-M - which you'll most likely not. [The introduction of this guide](http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-7986/nrf/ug_nrf91_features.html#application) also states this: «The user application runs in the non-secure domain. Therefore, it must be built for the nrf9160dk_nrf9160_ns or thingy91_nrf9160_ns build target.».

So please remove `nrf9160dk_nrf9160`.